site/content/device/pc-cooling-fan.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "PC Cooling Fan"
+description = "Move air across to cool a embedded components."
+picture = "/img/pc-cooling-fan.jpg"
++++
+
+# Overview
+[GPIO pins](/device/gpio/) can be leveraged to send pulses and read presses, similar to
+[buzzer](/device/buzzer/) and the [detecting motion](/device/pir/).
+
+Pulses can be sent to a fan to start or stop ventilation.
+
+## Current
+
+Fans can vary a lot, and have different volts and amps and somtimes can be too much for a GPIO.
```

Comment:
sometimes

But I think that's the primary point you wanted to expand on. You could talk about [L2930](http://www.ti.com/lit/ds/symlink/l293.pdf) which is useful for this kind of functionality. Here's an example drawing: https://github.com/maruel/bolide-esp8266#wiring

Reply:
Can you provide more description